src/reanimated2/layoutReanimation/animationBuilder/commonTypes.ts

```diff
@@ -79,7 +79,7 @@ export type LayoutAnimationFunction = (
 export type LayoutAnimationStartFunction = (
   tag: number,
   type: LayoutAnimationType,
-  yogaValues: LayoutAnimationsValues,
+  yogaValues: Partial<SharedTransitionAnimationsValues>,
```

Comment:
Why did these become `SharedTransitionAnimationsValues`?

Reply:
Well, the answer is:
1. We don't use type `LayoutAnimationStartFunction` at all, we just export it
2. This type is equal to input arguments of function ` createLayoutAnimationManager`, which works with both layout and shared transition tag animations. So once I've updated `createLayoutAnimationManager` I've updated type `LayoutAnimationStartFunction` to keep then consistent.